doc/nrf/protocols/matter/end_product/security.rst

```diff
@@ -129,6 +129,81 @@ This is a reference configuration that can be modified in the production firmwar
+   * - nRF5340 SoC + nRF7002 companion IC
+     - Not available
+     - Not available
+   * - nRF54L15 SoC
+     - Trusted Storage library + Hardware Unique Key (HUK)
+     - Key Management Unit (KMU), Trusted Storage library + Hardware Unique Key (HUK),  Trusted Storage library + SHA-256 hash
+   * - nRF54L15 SoC + Trusted Firmware-M (TF-M)
+     - Trusted Storage library + Hardware Unique Key (HUK)
+     - Key Management Unit (KMU), Trusted Firmware-M (TF-M) Storage
+
+.. _matter_platforms_security_dac_priv_key_its:
+
+DAC in Trusted Storage library
+==============================
+
+The Device Attestation Certificates private key can be stored in the Trusted Storage library.
+The key is encrypted with the AEAD key derived from the Hardware Unique Key (HUK) or a SHA-256 hash.
+This storage backend is selected by default for all platforms that support the PSA crypto API.
+
+To enable storing the DAC private key in the Trusted Storage library, set the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA_DAC_PRIV_KEY_ITS` Kconfig option to ``y``.
+To select encryption choose one of the possible options:
+
+- To use key derivation from HUK, set the :kconfig:option:`CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_KEY_DERIVE_FROM_HUK` Kconfig option to ``y``.
+- To use key derivation from a SHA-256 hash, set the :kconfig:option:`CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_KEY_HASH_UID` Kconfig option to ``y``.
+
+Encryption with the AEAD key derived from the Hardware Unique Key (HUK) is available only on the nRF5340 and nRF54L platforms.
+
+.. _matter_platforms_security_dac_priv_key_kmu:
+
+DAC in Key Management Unit (KMU)
+================================
+
+The Key Management Unit (KMU) is a hardware peripheral that provides secure storage for cryptographic keys.
+It is available in the nRF54L SoC series and can be used to store the DAC private key.
+This storage backend can be used with Trusted Firmware-M (TF-M).
+
+You can enable storing the DAC private key in the KMU by setting the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA_DAC_PRIV_KEY_KMU` Kconfig option to ``y``.
+
+You can additionally encrypt the DAC private key in the KMU storage by setting the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA_DAC_PRIV_KEY_KMU_ENCRYPTED` Kconfig option to ``y``.
+This operation requires two additional KMU slots to store the nonce and the authentication tag, making the total number of slots used four.
+If the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA_DAC_PRIV_KEY_KMU_ENCRYPTED` Kconfig option is set to ``n``, then the DAC private key is stored in the KMU without encryption and utilizes two KMU slots.
+
+By default, the DAC private key occupies the last slots dedicated for application purposes.
+For the non-encrypted version, it occupies slots 178 and 179, and for the encrypted version, it occupies slots 176-179.
+You can change the default slots by setting the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA_DAC_PRIV_KEY_KMU_SLOT` Kconfig option to the desired slot number within the possible range of 0-179.
```

Comment:
Is the defined slot the first or the last slot? If it's the last, it can't be 0, and if it's the first, it can't be 179.

Reply:
The possible range that the user can use to store this key is 0-179, whereas we use the last 4 slots (176,177,178,179) for Encrypted, and the last 2 slots for not-encrypted version. Should I explain it better? :)

Reply:
Ok, maybe it would be better to write that for encryption available slots are 0-176 and without it 0-178. :)